tests/system-tests/rdscore/internal/rdscorecommon/metallb-traffic-segregation.go

```diff
@@ -0,0 +1,731 @@
+			RDSCoreConfig.HypervisorUser,
+			RDSCoreConfig.HypervisorPass,
+			RDSCoreConfig.MetalLBFRRContainerNameTwo,
+			ipRoute)
+		Expect(err).ToNot(HaveOccurred(),
+			fmt.Sprintf("second external FRR container %s route check for the %s failed: %v",
+				RDSCoreConfig.MetalLBFRRContainerNameTwo, ipRoute, err))
+	}
+
+	By("Assert that 1st mockup app is not reachable from the node with 2st FRR container")
+
+	timeStart := time.Now()
+
+	glog.V(rdscoreparams.RDSCoreLogLevel).Infof("start run traffic at %v", timeStart)
+
+	for _, appURL := range []string{bgpOneAppURLIPv4, bgpOneAppURLIPv6} {
+		err := verifyAppIsReachableFromFRRContainer(
+			RDSCoreConfig.HypervisorHost,
+			RDSCoreConfig.HypervisorUser,
+			RDSCoreConfig.HypervisorPass,
+			RDSCoreConfig.MetalLBFRRContainerNameTwo,
+			appURL)
+		Expect(err).To(HaveOccurred(),
+			fmt.Sprintf("first mockup app %s is reachable from the container %s: %v",
+				appURL, RDSCoreConfig.MetalLBFRRContainerNameTwo, err))
+	}
+
+	By("Assert that 2nd mockup app is reachable from the node with 2nd FRR container")
+
+	for _, appURL := range []string{bgpTwoAppURLIPv4, bgpTwoAppURLIPv6} {
+		err := verifyAppIsReachableFromFRRContainer(
+			RDSCoreConfig.HypervisorHost,
+			RDSCoreConfig.HypervisorUser,
+			RDSCoreConfig.HypervisorPass,
+			RDSCoreConfig.MetalLBFRRContainerNameOne,
+			appURL)
+		Expect(err).To(HaveOccurred(),
+			fmt.Sprintf("second mockup app %s is not reachable from the container %s: %v",
+				appURL, RDSCoreConfig.MetalLBFRRContainerNameOne, err))
+	}
+}
+
+func verifyIPRouteBGP(host, user, pass, containerName, lbIP string) error {
+	var result string
+
+	var err error
+
+	glog.V(100).Infof("Verify IP version type")
+
+	parsedIP := strings.Split(lbIP, "/")[0]
+
+	glog.V(100).Infof("IP: %s", parsedIP)
+
+	myIP, err := netip.ParseAddr(parsedIP)
+
+	if err != nil {
+		glog.V(100).Infof("Failed to parse provided loadbalancer ip address %q; %v", parsedIP, err)
+
+		return fmt.Errorf("failed to parse provided loadbalancer ip address %q; %w", parsedIP, err)
+	}
+
+	showIPRouteBGPOnFRRContainerCmd := fmt.Sprintf("sudo podman exec -it %s bash "+
+		"-c 'vtysh -c \"show ip route bgp\"'", containerName)
+
+	if myIP.Is6() {
+		showIPRouteBGPOnFRRContainerCmd = fmt.Sprintf("sudo podman exec -it %s bash "+
+			"-c 'vtysh -c \"show ipv6 route bgp\"'", containerName)
+	}
+
+	glog.V(100).Infof("Running command %q from within container %s",
+		showIPRouteBGPOnFRRContainerCmd, containerName)
+
+	err = wait.PollUntilContextTimeout(
+		context.TODO(),
+		time.Second,
+		time.Second*30,
+		true,
+		func(ctx context.Context) (bool, error) {
+			result, err = remote.ExecCmdOnHost(host, user, pass, showIPRouteBGPOnFRRContainerCmd)
+
+			if err != nil {
+				glog.V(rdscoreparams.RDSCoreLogLevel).Infof("Failed to run command due to: %v", err)
+
+				return false, nil
+			}
+
+			glog.V(100).Infof(fmt.Sprintf("the IP routes %s were found on the FRR container %s: %s",
+				lbIP, containerName, result))
```

Comment:
the command is `show ip(v6) route bgp` so not sure why it's stated that route for requested IP address was present in the output of the aforementioned command:
```suggestion
			glog.V(100).Infof(fmt.Sprintf("BGP routes were found in the FRR container %s\n: %s",
			    containerName, result))
```